seed/views/v3/organizations.py

```diff
@@ -1392,3 +1403,353 @@ def reset_all_passwords(self, request, pk=None):
+            'year_built': 1988,
+            'site_eui': 54.7,
+            'gross_floor_area': 373152,
+            'owner': 'Landmark Network',
+            'owner_email': 'landmark@gmail.com',
+            'owner_telephone': '555-555-5563',
+            'owner_address': '159308 SE Sycamore Court',
+            'owner_city_state': 'Colorado',
+            'owner_postal_code': 80202,
+            'organization_id': pk,
+            'extra_data': {'Note': 'Residential'}
+        }, {
+            'jurisdiction_property_id': 123465,
+            'pm_parent_property_id': '5766979',
+            'pm_property_id': '5766979',
+            'lot_number': 'J-12354',
+            'address_line_1': '180702 N Horsechestnut Loop',
+            'city': 'Boring',
+            'state': 'Oregon',
+            'postal_code': 80401,
+            'year_built': 2000,
+            'site_eui': 31.4,
+            'gross_floor_area': 373152,
+            'owner': 'Memphis Invest',
+            'owner_email': 'memphis.invest@gmail.com',
+            'owner_telephone': '555-555-5564',
+            'owner_address': '94734 SE Honeylocust Street',
+            'owner_city_state': 'Colorado',
+            'owner_postal_code': 80202,
+            'organization_id': pk
+        }, {
+            'jurisdiction_property_id': 123465,
+            'pm_parent_property_id': '5766979',
+            'pm_property_id': '5766979',
+            'lot_number': 'J-12354',
+            'address_line_1': '180702 N Horsechestnut Loop',
+            'city': 'Boring',
+            'state': 'Oregon',
+            'postal_code': 80401,
+            'year_built': 2000,
+            'site_eui': 45.6,
+            'gross_floor_area': 373152,
+            'owner': 'Memphis Invest',
+            'owner_email': 'memphis.invest@gmail.com',
+            'owner_telephone': '555-555-5564',
+            'owner_address': '94734 SE Honeylocust Street',
+            'owner_city_state': 'Colorado',
+            'owner_postal_code': 80202,
+            'organization_id': pk
+        }]
+
+        taxlot_details = {
+            'jurisdiction_tax_lot_id': 'A-12345',
+            'city': 'Boring',
+            'organization_id': pk,
+            'extra_data': {'Note': 'This is my first note'}
+        }
+
+        taxlot_state = TaxLotState(**taxlot_details)
+        taxlot_state.save()
+        taxlot_1 = TaxLot.objects.create(organization=org)
+        taxview = TaxLotView.objects.create(taxlot=taxlot_1, cycle=cycle, state=taxlot_state)
+
+        state_ids_to_merge = []
+        property_views = []
+        properties = []
+        ids = []
+        for dic in property_details:
+            state = PropertyState(**dic)
+            state.save()
+            ids.append(state.id)
+
+            # merge
+            for count in range(len(property_details)):
+                if count > 9:
+                    state_ids_to_merge = [state.id]
+                    state_ids_to_merge.append(state.id)
```

Comment:
[non-critical] This is a bit confusing. Could we break this loop out of here so we construct state_ids_to_merge after creating all of the property states? It looks like we want to merge states 10 and onwards, so something like this might be a bit more clear
```
# after creating all states
state_ids_to_merge = [id for id in ids[9:]]
```